{"occurAt":"12/10/2020 3:47:10 PM","exceptionDetails":{"ErrorMessage":"Object reference not set to an instance of an object.","OriginatingClass":"medicloud.emr.api.Controllers.PatientController+<RegisterNewPatient>d__13","FileName":"C:\\Users\\raimi.aliu\\Desktop\\tunde\\PROJECTS\\medismartprojects\\medicloud-emr-api\\medicloud.emr.api\\Controllers\\PatientController.cs","LineNumber":230,"ColumnNumber":17,"MethodThatThrowException":"Void MoveNext()"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/patient/newpatient","ControllerMethodName":"RegisterNewPatient","ControllerName":"Patient"}}

{"occurAt":"12/10/2020 3:55:09 PM","exceptionDetails":{"ErrorMessage":"An error occurred while updating the entries. See the inner exception for details.","OriginatingClass":"medicloud.emr.api.Services.PatientService","FileName":"C:\\Users\\raimi.aliu\\Desktop\\tunde\\PROJECTS\\medismartprojects\\medicloud-emr-api\\medicloud.emr.api\\Services\\IPatientServices.cs","LineNumber":44,"ColumnNumber":17,"MethodThatThrowException":"System.String addNewPatient(medicloud.emr.api.DTOs.PatientDTO)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/patient/newpatient","ControllerMethodName":"RegisterNewPatient","ControllerName":"Patient"}}
{"occurAt":"12/10/2020 4:33:05 PM","exceptionDetails":{"ErrorMessage":"An error occurred while updating the entries. See the inner exception for details.","OriginatingClass":"medicloud.emr.api.Services.PatientService","FileName":"C:\\Users\\raimi.aliu\\Desktop\\tunde\\PROJECTS\\medismartprojects\\medicloud-emr-api\\medicloud.emr.api\\Services\\IPatientServices.cs","LineNumber":44,"ColumnNumber":17,"MethodThatThrowException":"System.String addNewPatient(medicloud.emr.api.DTOs.PatientDTO)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/patient/newpatient","ControllerMethodName":"RegisterNewPatient","ControllerName":"Patient"}}
{"occurAt":"12/10/2020 5:52:35 PM","exceptionDetails":{"ErrorMessage":"An error occurred while updating the entries. See the inner exception for details.","OriginatingClass":"Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void Execute(Microsoft.EntityFrameworkCore.Storage.IRelationalConnection)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/patient/saveRegistrationLink/10897409549","ControllerMethodName":"SaveRegistrationLink","ControllerName":"Patient"}}
{"occurAt":"12/10/2020 7:26:03 PM","exceptionDetails":{"ErrorMessage":"The instance of entity type 'Patient' cannot be tracked because another instance with the same key value for {'Patientid'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.","OriginatingClass":"Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void ThrowIdentityConflict(Microsoft.EntityFrameworkCore.ChangeTracking.Internal.InternalEntityEntry)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/patient/registerPatientFromLink/5977944792","ControllerMethodName":"registerPatientFromLink","ControllerName":"Patient"}}
{"occurAt":"12/10/2020 7:27:17 PM","exceptionDetails":{"ErrorMessage":"The instance of entity type 'Patient' cannot be tracked because another instance with the same key value for {'Patientid'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.","OriginatingClass":"Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void ThrowIdentityConflict(Microsoft.EntityFrameworkCore.ChangeTracking.Internal.InternalEntityEntry)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/patient/registerPatientFromLink/5977944792","ControllerMethodName":"registerPatientFromLink","ControllerName":"Patient"}}
{"occurAt":"12/17/2020 9:51:52 AM","exceptionDetails":{"ErrorMessage":"Unable to cast object of type 'System.DateTime' to type 'System.String'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlBuffer","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.String get_String()"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/setup/allBloodGroup","ControllerMethodName":"AllBloodGroup","ControllerName":"Setup"}}
{"occurAt":"12/17/2020 10:00:46 AM","exceptionDetails":{"ErrorMessage":"Unable to cast object of type 'System.DateTime' to type 'System.String'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlBuffer","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.String get_String()"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/setup/allBloodGroup","ControllerMethodName":"AllBloodGroup","ControllerName":"Setup"}}
{"occurAt":"12/17/2020 10:05:39 AM","exceptionDetails":{"ErrorMessage":"Unable to cast object of type 'System.DateTime' to type 'System.String'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlBuffer","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.String get_String()"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/setup/allBloodGroup","ControllerMethodName":"AllBloodGroup","ControllerName":"Setup"}}
{"occurAt":"12/17/2020 10:13:39 AM","exceptionDetails":{"ErrorMessage":"Unable to cast object of type 'System.DateTime' to type 'System.String'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlBuffer","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.String get_String()"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/setup/allBloodGroup","ControllerMethodName":"AllBloodGroup","ControllerName":"Setup"}}
{"occurAt":"12/17/2020 11:15:54 AM","exceptionDetails":{"ErrorMessage":"Invalid object name 'TemplateCategoryB'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/form","ControllerMethodName":"GetData","ControllerName":"Form"}}
{"occurAt":"12/17/2020 11:16:57 AM","exceptionDetails":{"ErrorMessage":"Invalid object name 'TemplateCategoryB'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/form","ControllerMethodName":"GetData","ControllerName":"Form"}}
{"occurAt":"12/17/2020 11:17:14 AM","exceptionDetails":{"ErrorMessage":"Invalid object name 'TemplateCategoryB'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/form","ControllerMethodName":"GetData","ControllerName":"Form"}}
{"occurAt":"12/17/2020 11:34:36 AM","exceptionDetails":{"ErrorMessage":"Invalid object name 'TemplateCategoryB'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/form","ControllerMethodName":"GetData","ControllerName":"Form"}}
{"occurAt":"12/17/2020 11:36:45 AM","exceptionDetails":{"ErrorMessage":"The request matched multiple endpoints. Matches: \r\n\r\nmedicloud.emr.api.Controllers.FormController.GetSingleForm (medicloud.emr.api)\r\nmedicloud.emr.api.Controllers.FormController.GetFormByName (medicloud.emr.api)","OriginatingClass":"Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void ReportAmbiguity(Microsoft.AspNetCore.Routing.Matching.CandidateState[])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/form/allforms","ControllerMethodName":"","ControllerName":""}}
{"occurAt":"12/17/2020 12:17:50 PM","exceptionDetails":{"ErrorMessage":"One or more errors occurred. (A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.)","OriginatingClass":"medicloud.emr.api.Services.PatientService","FileName":"C:\\Users\\raimi.aliu\\Desktop\\tunde\\PROJECTS\\medismartprojects\\medicloud-emr-api\\medicloud.emr.api\\Services\\IPatientServices.cs","LineNumber":44,"ColumnNumber":17,"MethodThatThrowException":"System.String addNewPatient(medicloud.emr.api.DTOs.PatientDTO)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/Patient/newPatient","ControllerMethodName":"RegisterNewPatient","ControllerName":"Patient"}}
{"occurAt":"12/17/2020 12:24:07 PM","exceptionDetails":{"ErrorMessage":"One or more errors occurred. (A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.)","OriginatingClass":"medicloud.emr.api.Services.PatientService","FileName":"C:\\Users\\raimi.aliu\\Desktop\\tunde\\PROJECTS\\medismartprojects\\medicloud-emr-api\\medicloud.emr.api\\Services\\IPatientServices.cs","LineNumber":44,"ColumnNumber":17,"MethodThatThrowException":"System.String addNewPatient(medicloud.emr.api.DTOs.PatientDTO)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/Patient/newPatient","ControllerMethodName":"RegisterNewPatient","ControllerName":"Patient"}}
{"occurAt":"12/17/2020 12:28:31 PM","exceptionDetails":{"ErrorMessage":"Misused header name. Make sure request headers are used with HttpRequestMessage, response headers with HttpResponseMessage, and content headers with HttpContent objects.","OriginatingClass":"medicloud.emr.api.Services.PatientService","FileName":"C:\\Users\\raimi.aliu\\Desktop\\tunde\\PROJECTS\\medismartprojects\\medicloud-emr-api\\medicloud.emr.api\\Services\\IPatientServices.cs","LineNumber":44,"ColumnNumber":17,"MethodThatThrowException":"System.String addNewPatient(medicloud.emr.api.DTOs.PatientDTO)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/Patient/newPatient","ControllerMethodName":"RegisterNewPatient","ControllerName":"Patient"}}
{"occurAt":"12/17/2020 12:42:14 PM","exceptionDetails":{"ErrorMessage":"One or more errors occurred. (A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.)","OriginatingClass":"medicloud.emr.api.Services.PatientService","FileName":"C:\\Users\\raimi.aliu\\Desktop\\tunde\\PROJECTS\\medismartprojects\\medicloud-emr-api\\medicloud.emr.api\\Services\\IPatientServices.cs","LineNumber":44,"ColumnNumber":17,"MethodThatThrowException":"System.String addNewPatient(medicloud.emr.api.DTOs.PatientDTO)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/Patient/newPatient","ControllerMethodName":"RegisterNewPatient","ControllerName":"Patient"}}
{"occurAt":"12/17/2020 12:57:49 PM","exceptionDetails":{"ErrorMessage":"One or more errors occurred. (A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.)","OriginatingClass":"medicloud.emr.api.Services.PatientService","FileName":"C:\\Users\\raimi.aliu\\Desktop\\tunde\\PROJECTS\\medismartprojects\\medicloud-emr-api\\medicloud.emr.api\\Services\\IPatientServices.cs","LineNumber":44,"ColumnNumber":17,"MethodThatThrowException":"System.String addNewPatient(medicloud.emr.api.DTOs.PatientDTO)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/Patient/newPatient","ControllerMethodName":"RegisterNewPatient","ControllerName":"Patient"}}
{"occurAt":"12/17/2020 1:07:02 PM","exceptionDetails":{"ErrorMessage":"Misused header name. Make sure request headers are used with HttpRequestMessage, response headers with HttpResponseMessage, and content headers with HttpContent objects.","OriginatingClass":"medicloud.emr.api.Services.PatientService","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.String addNewPatient(medicloud.emr.api.DTOs.PatientDTO)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/Patient/newPatient","ControllerMethodName":"RegisterNewPatient","ControllerName":"Patient"}}
{"occurAt":"12/17/2020 1:19:21 PM","exceptionDetails":{"ErrorMessage":"error generating patientId from Akhil endpoint","OriginatingClass":"medicloud.emr.api.Services.PatientService","FileName":"C:\\Users\\raimi.aliu\\Desktop\\tunde\\PROJECTS\\medismartprojects\\medicloud-emr-api\\medicloud.emr.api\\Services\\IPatientServices.cs","LineNumber":44,"ColumnNumber":17,"MethodThatThrowException":"System.String addNewPatient(medicloud.emr.api.DTOs.PatientDTO)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/Patient/newPatient","ControllerMethodName":"RegisterNewPatient","ControllerName":"Patient"}}
{"occurAt":"12/17/2020 1:21:56 PM","exceptionDetails":{"ErrorMessage":"error generating patientId from Akhil endpoint","OriginatingClass":"medicloud.emr.api.Services.PatientService","FileName":"C:\\Users\\raimi.aliu\\Desktop\\tunde\\PROJECTS\\medismartprojects\\medicloud-emr-api\\medicloud.emr.api\\Services\\IPatientServices.cs","LineNumber":44,"ColumnNumber":17,"MethodThatThrowException":"System.String addNewPatient(medicloud.emr.api.DTOs.PatientDTO)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/Patient/newPatient","ControllerMethodName":"RegisterNewPatient","ControllerName":"Patient"}}
{"occurAt":"12/17/2020 1:33:40 PM","exceptionDetails":{"ErrorMessage":"An error occurred while updating the entries. See the inner exception for details.","OriginatingClass":"medicloud.emr.api.Services.PatientService","FileName":"C:\\Users\\raimi.aliu\\Desktop\\tunde\\PROJECTS\\medismartprojects\\medicloud-emr-api\\medicloud.emr.api\\Services\\IPatientServices.cs","LineNumber":44,"ColumnNumber":17,"MethodThatThrowException":"System.String addNewPatient(medicloud.emr.api.DTOs.PatientDTO)"},"RequestDetails":{"HttpMethod":"POST","Path":"/api/Patient/newPatient","ControllerMethodName":"RegisterNewPatient","ControllerName":"Patient"}}
{"occurAt":"12/22/2020 10:24:15","exceptionDetails":{"ErrorMessage":"Connection Timeout Expired.  The timeout period elapsed while attempting to consume the pre-login handshake acknowledgement.  This could be because the pre-login handshake failed or the server was unable to respond back in time.  The duration spent while attempting to connect to this server was - [Pre-Login] initialization=16150; handshake=4; ","OriginatingClass":"Microsoft.Data.SqlClient.SqlInternalConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/CheckIn/GetCheckedInPatient","ControllerMethodName":"GetCheckedInPatient","ControllerName":"CheckIn"}}
{"occurAt":"12/22/2020 15:02:07","exceptionDetails":{"ErrorMessage":"Ambiguous HTTP method for action - medicloud.emr.api.Controllers.FormController.CreateTable (medicloud.emr.api). Actions require an explicit HttpMethod binding for Swagger/OpenAPI 3.0","OriginatingClass":"Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Collections.Generic.IDictionary`2[Microsoft.OpenApi.Models.OperationType,Microsoft.OpenApi.Models.OpenApiOperation] GenerateOperations(System.Collections.Generic.IEnumerable`1[Microsoft.AspNetCore.Mvc.ApiExplorer.ApiDescription], Swashbuckle.AspNetCore.SwaggerGen.SchemaRepository)"},"RequestDetails":{"HttpMethod":"GET","Path":"/swagger/v1/swagger.json","ControllerMethodName":"","ControllerName":""}}
{"occurAt":"12/22/2020 15:24:11","exceptionDetails":{"ErrorMessage":"Ambiguous HTTP method for action - medicloud.emr.api.Controllers.FormController.CreateTable (medicloud.emr.api). Actions require an explicit HttpMethod binding for Swagger/OpenAPI 3.0","OriginatingClass":"Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Collections.Generic.IDictionary`2[Microsoft.OpenApi.Models.OperationType,Microsoft.OpenApi.Models.OpenApiOperation] GenerateOperations(System.Collections.Generic.IEnumerable`1[Microsoft.AspNetCore.Mvc.ApiExplorer.ApiDescription], Swashbuckle.AspNetCore.SwaggerGen.SchemaRepository)"},"RequestDetails":{"HttpMethod":"GET","Path":"/swagger/v1/swagger.json","ControllerMethodName":"","ControllerName":""}}
{"occurAt":"12/22/2020 15:28:37","exceptionDetails":{"ErrorMessage":"Ambiguous HTTP method for action - medicloud.emr.api.Controllers.FormController.CreateTable (medicloud.emr.api). Actions require an explicit HttpMethod binding for Swagger/OpenAPI 3.0","OriginatingClass":"Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Collections.Generic.IDictionary`2[Microsoft.OpenApi.Models.OperationType,Microsoft.OpenApi.Models.OpenApiOperation] GenerateOperations(System.Collections.Generic.IEnumerable`1[Microsoft.AspNetCore.Mvc.ApiExplorer.ApiDescription], Swashbuckle.AspNetCore.SwaggerGen.SchemaRepository)"},"RequestDetails":{"HttpMethod":"GET","Path":"/swagger/v1/swagger.json","ControllerMethodName":"","ControllerName":""}}
{"occurAt":"12/22/2020 15:37:24","exceptionDetails":{"ErrorMessage":"Ambiguous HTTP method for action - medicloud.emr.api.Controllers.FormController.CreateTable (medicloud.emr.api). Actions require an explicit HttpMethod binding for Swagger/OpenAPI 3.0","OriginatingClass":"Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Collections.Generic.IDictionary`2[Microsoft.OpenApi.Models.OperationType,Microsoft.OpenApi.Models.OpenApiOperation] GenerateOperations(System.Collections.Generic.IEnumerable`1[Microsoft.AspNetCore.Mvc.ApiExplorer.ApiDescription], Swashbuckle.AspNetCore.SwaggerGen.SchemaRepository)"},"RequestDetails":{"HttpMethod":"GET","Path":"/swagger/v1/swagger.json","ControllerMethodName":"","ControllerName":""}}
{"occurAt":"12/22/2020 16:43:28","exceptionDetails":{"ErrorMessage":"Ambiguous HTTP method for action - medicloud.emr.api.Controllers.FormController.CreateTable (medicloud.emr.api). Actions require an explicit HttpMethod binding for Swagger/OpenAPI 3.0","OriginatingClass":"Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Collections.Generic.IDictionary`2[Microsoft.OpenApi.Models.OperationType,Microsoft.OpenApi.Models.OpenApiOperation] GenerateOperations(System.Collections.Generic.IEnumerable`1[Microsoft.AspNetCore.Mvc.ApiExplorer.ApiDescription], Swashbuckle.AspNetCore.SwaggerGen.SchemaRepository)"},"RequestDetails":{"HttpMethod":"GET","Path":"/swagger/v1/swagger.json","ControllerMethodName":"","ControllerName":""}}
{"occurAt":"12/23/2020 04:29:03","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.SqlClient.SqlInternalConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/CheckIn/GetCheckedInPatient","ControllerMethodName":"GetCheckedInPatient","ControllerName":"CheckIn"}}
{"occurAt":"12/23/2020 13:32:12","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/appointments/statuses","ControllerMethodName":"GetAppointmentStatuses","ControllerName":"Appointments"}}
{"occurAt":"12/23/2020 13:32:14","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.SqlClient.SqlInternalConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations","ControllerMethodName":"GetLocations","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 13:49:16","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.SqlClient.SqlInternalConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/CheckIn/GetCheckedInPatient","ControllerMethodName":"GetCheckedInPatient","ControllerName":"CheckIn"}}
{"occurAt":"12/23/2020 13:49:57","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations","ControllerMethodName":"GetLocations","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 13:50:24","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/CheckIn/GetCheckedInPatient","ControllerMethodName":"GetCheckedInPatient","ControllerName":"CheckIn"}}
{"occurAt":"12/23/2020 13:50:24","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations","ControllerMethodName":"GetLocations","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 13:50:24","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/appointments/statuses","ControllerMethodName":"GetAppointmentStatuses","ControllerName":"Appointments"}}
{"occurAt":"12/23/2020 13:50:24","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/appointments/statuses","ControllerMethodName":"GetAppointmentStatuses","ControllerName":"Appointments"}}
{"occurAt":"12/23/2020 13:50:38","exceptionDetails":{"ErrorMessage":"Invalid column name 'Accountid'.\r\nInvalid column name 'Accountid'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlCommand+<>c","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/CheckIn/GetCheckedInPatient","ControllerMethodName":"GetCheckedInPatient","ControllerName":"CheckIn"}}
{"occurAt":"12/23/2020 14:02:55","exceptionDetails":{"ErrorMessage":"Ambiguous HTTP method for action - medicloud.emr.api.Controllers.FormController.CreateTable (medicloud.emr.api). Actions require an explicit HttpMethod binding for Swagger/OpenAPI 3.0","OriginatingClass":"Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Collections.Generic.IDictionary`2[Microsoft.OpenApi.Models.OperationType,Microsoft.OpenApi.Models.OpenApiOperation] GenerateOperations(System.Collections.Generic.IEnumerable`1[Microsoft.AspNetCore.Mvc.ApiExplorer.ApiDescription], Swashbuckle.AspNetCore.SwaggerGen.SchemaRepository)"},"RequestDetails":{"HttpMethod":"GET","Path":"/swagger/v1/swagger.json","ControllerMethodName":"","ControllerName":""}}
{"occurAt":"12/23/2020 14:45:00","exceptionDetails":{"ErrorMessage":"Invalid column name 'Reminder1'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlCommand+<>c","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/reminderoptions","ControllerMethodName":"GetReminderOptions","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 14:52:57","exceptionDetails":{"ErrorMessage":"Invalid column name 'Reminder1'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlCommand+<>c","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/reminderoptions","ControllerMethodName":"GetReminderOptions","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 14:53:47","exceptionDetails":{"ErrorMessage":"Invalid column name 'Reminder1'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlCommand+<>c","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/reminderoptions","ControllerMethodName":"GetReminderOptions","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 15:05:08","exceptionDetails":{"ErrorMessage":"Invalid column name 'Reminder1'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlCommand+<>c","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/reminderoptions","ControllerMethodName":"GetReminderOptions","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 15:08:53","exceptionDetails":{"ErrorMessage":"Invalid column name 'Reminder1'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlCommand+<>c","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/reminderoptions","ControllerMethodName":"GetReminderOptions","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 15:13:00","exceptionDetails":{"ErrorMessage":"Invalid column name 'Reminder1'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlCommand+<>c","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/reminderoptions","ControllerMethodName":"GetReminderOptions","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 15:26:52","exceptionDetails":{"ErrorMessage":"A transport-level error has occurred when receiving results from the server. (provider: Session Provider, error: 19 - Physical connection is not usable)","OriginatingClass":"Microsoft.Data.SqlClient.SqlCommand+<>c","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/0/specializations/0/providers","ControllerMethodName":"GetProviders","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 15:27:10","exceptionDetails":{"ErrorMessage":"A transport-level error has occurred when receiving results from the server. (provider: Session Provider, error: 19 - Physical connection is not usable)","OriginatingClass":"Microsoft.Data.SqlClient.SqlCommand+<>c","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/0/specializations","ControllerMethodName":"GetSpecializations","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 15:27:10","exceptionDetails":{"ErrorMessage":"A transport-level error has occurred when receiving results from the server. (provider: Session Provider, error: 19 - Physical connection is not usable)","OriginatingClass":"Microsoft.Data.SqlClient.SqlCommand+<>c","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations","ControllerMethodName":"GetLocations","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 15:27:11","exceptionDetails":{"ErrorMessage":"A transport-level error has occurred when receiving results from the server. (provider: Session Provider, error: 19 - Physical connection is not usable)","OriginatingClass":"Microsoft.Data.SqlClient.SqlCommand+<>c","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/appointments/statuses","ControllerMethodName":"GetAppointmentStatuses","ControllerName":"Appointments"}}
{"occurAt":"12/23/2020 15:27:18","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/referringphysicians","ControllerMethodName":"GetReferringPhysicians","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 15:27:55","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.SqlClient.SqlInternalConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/visittypes","ControllerMethodName":"GetVisitTypes","ControllerName":"Locations"}}
{"occurAt":"12/23/2020 15:39:27","exceptionDetails":{"ErrorMessage":"Unable to cast object of type 'System.DateTime' to type 'System.String'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlBuffer","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.String get_String()"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/setup/allBloodGroup","ControllerMethodName":"AllBloodGroup","ControllerName":"Setup"}}
{"occurAt":"12/23/2020 15:39:28","exceptionDetails":{"ErrorMessage":"Invalid column name 'IsinsuranceCompany'.\r\nInvalid column name 'PayerType'.","OriginatingClass":"Microsoft.Data.SqlClient.SqlConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/setup/getAllPayer","ControllerMethodName":"getAllPayer","ControllerName":"Setup"}}
{"occurAt":"12/23/2020 15:39:29","exceptionDetails":{"ErrorMessage":"Data is Null. This method or property cannot be called on Null values.","OriginatingClass":"Microsoft.Data.SqlClient.SqlBuffer","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void ThrowIfNull()"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/setup/getAllSponsor","ControllerMethodName":"getAllSponsor","ControllerName":"Setup"}}
{"occurAt":"12/23/2020 18:46:14","exceptionDetails":{"ErrorMessage":"The property or navigation 'Patient' cannot be added to the entity type 'AppointmentSchedule' because a property or navigation with the same name already exists on entity type 'AppointmentSchedule'.","OriginatingClass":"Microsoft.EntityFrameworkCore.Metadata.Internal.EntityType","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Microsoft.EntityFrameworkCore.Metadata.Internal.Navigation AddNavigation(Microsoft.EntityFrameworkCore.Metadata.MemberIdentity, Microsoft.EntityFrameworkCore.Metadata.Internal.ForeignKey, Boolean)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/CheckIn/GetCheckedInPatient","ControllerMethodName":"GetCheckedInPatient","ControllerName":"CheckIn"}}
{"occurAt":"12/23/2020 19:16:17","exceptionDetails":{"ErrorMessage":"A connection was successfully established with the server, but then an error occurred during the pre-login handshake. (provider: SSL Provider, error: 0 - The wait operation timed out.)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations","ControllerMethodName":"GetLocations","ControllerName":"Locations"}}
{"occurAt":"12/24/2020 12:51:44","exceptionDetails":{"ErrorMessage":"Execution Timeout Expired.  The timeout period elapsed prior to completion of the operation or the server is not responding.","OriginatingClass":"Microsoft.Data.SqlClient.SqlInternalConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"PUT","Path":"/api/appointments/update/1003","ControllerMethodName":"UpdateAppointment","ControllerName":"Appointments"}}
{"occurAt":"12/24/2020 12:51:53","exceptionDetails":{"ErrorMessage":"A transport-level error has occurred when receiving results from the server. (provider: Session Provider, error: 19 - Physical connection is not usable)","OriginatingClass":"Microsoft.Data.SqlClient.SqlCommand+<>c","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader])"},"RequestDetails":{"HttpMethod":"PUT","Path":"/api/appointments/update/1003","ControllerMethodName":"UpdateAppointment","ControllerName":"Appointments"}}
{"occurAt":"12/24/2020 12:52:50","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.SqlClient.SqlInternalConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"PUT","Path":"/api/appointments/update/1003","ControllerMethodName":"UpdateAppointment","ControllerName":"Appointments"}}
{"occurAt":"12/24/2020 12:55:35","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/referraltypes","ControllerMethodName":"GetReferralTypes","ControllerName":"Locations"}}
{"occurAt":"12/24/2020 12:55:36","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/4/specializations/2/providers","ControllerMethodName":"GetProviders","ControllerName":"Locations"}}
{"occurAt":"12/24/2020 12:55:36","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/visittypes","ControllerMethodName":"GetVisitTypes","ControllerName":"Locations"}}
{"occurAt":"12/24/2020 12:55:37","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations/referringphysicians","ControllerMethodName":"GetReferringPhysicians","ControllerName":"Locations"}}
{"occurAt":"12/24/2020 12:55:52","exceptionDetails":{"ErrorMessage":"Connection Timeout Expired.  The timeout period elapsed while attempting to consume the pre-login handshake acknowledgement.  This could be because the pre-login handshake failed or the server was unable to respond back in time.  The duration spent while attempting to connect to this server was - [Pre-Login] initialization=15301; handshake=0; ","OriginatingClass":"Microsoft.Data.SqlClient.SqlInternalConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/appointments/statuses","ControllerMethodName":"GetAppointmentStatuses","ControllerName":"Appointments"}}
{"occurAt":"12/26/2020 13:31:24","exceptionDetails":{"ErrorMessage":"There is not enough space on the disk. : 'C:\\Users\\JOEY'S\\AppData\\Local\\Temp\\ASPNETCORE_38dc656b-0e79-456f-9711-c50989bc4efd.tmp'","OriginatingClass":"System.IO.FileStream","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void SetLengthCore(Int64)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/Patient/getPatientBy/a","ControllerMethodName":"SearchForPatient","ControllerName":"Patient"}}
{"occurAt":"12/26/2020 13:31:27","exceptionDetails":{"ErrorMessage":"There is not enough space on the disk. : 'C:\\Users\\JOEY'S\\AppData\\Local\\Temp\\ASPNETCORE_18456b3c-fd63-4314-9bd6-b17025a8e7a3.tmp'","OriginatingClass":"System.IO.FileStream","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void SetLengthCore(Int64)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/Patient/getPatientBy/a","ControllerMethodName":"SearchForPatient","ControllerName":"Patient"}}
{"occurAt":"12/26/2020 13:36:57","exceptionDetails":{"ErrorMessage":"There is not enough space on the disk. : 'C:\\Users\\JOEY'S\\AppData\\Local\\Temp\\ASPNETCORE_d02e9f1e-f5ce-4d61-afa8-92663f918393.tmp'","OriginatingClass":"System.IO.FileStream","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void SetLengthCore(Int64)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/Patient/getPatientBy/a","ControllerMethodName":"SearchForPatient","ControllerName":"Patient"}}
{"occurAt":"12/26/2020 13:37:02","exceptionDetails":{"ErrorMessage":"There is not enough space on the disk. : 'C:\\Users\\JOEY'S\\AppData\\Local\\Temp\\ASPNETCORE_2decaef9-33df-4bd2-a820-ce872210d269.tmp'","OriginatingClass":"System.IO.FileStream","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void SetLengthCore(Int64)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/Patient/getPatientBy/a","ControllerMethodName":"SearchForPatient","ControllerName":"Patient"}}
{"occurAt":"12/26/2020 13:53:44","exceptionDetails":{"ErrorMessage":"There is not enough space on the disk. : 'C:\\Users\\JOEY'S\\AppData\\Local\\Temp\\ASPNETCORE_2ad8c912-9bb5-41e1-86b9-1ecfd774a2e5.tmp'","OriginatingClass":"System.IO.FileStream","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void SetLengthCore(Int64)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/Patient/getPatientBy/a","ControllerMethodName":"SearchForPatient","ControllerName":"Patient"}}
{"occurAt":"12/26/2020 13:57:01","exceptionDetails":{"ErrorMessage":"There is not enough space on the disk. : 'C:\\Users\\JOEY'S\\AppData\\Local\\Temp\\ASPNETCORE_97f8658f-45f7-4835-943a-6115376dab34.tmp'","OriginatingClass":"System.IO.FileStream","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void SetLengthCore(Int64)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/Patient/getPatientBy/a","ControllerMethodName":"SearchForPatient","ControllerName":"Patient"}}
{"occurAt":"12/26/2020 14:35:59","exceptionDetails":{"ErrorMessage":"There is not enough space on the disk. : 'C:\\Users\\JOEY'S\\AppData\\Local\\Temp\\ASPNETCORE_5f930db1-ff6c-4e71-a39b-e7064ee2ba58.tmp'","OriginatingClass":"System.IO.FileStream","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void SetLengthCore(Int64)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/Patient/getPatientBy/a","ControllerMethodName":"SearchForPatient","ControllerName":"Patient"}}
{"occurAt":"12/26/2020 16:50:19","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.SqlClient.SqlInternalConnection","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void OnError(Microsoft.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/CheckIn/GetCheckedInPatient","ControllerMethodName":"GetCheckedInPatient","ControllerName":"CheckIn"}}
{"occurAt":"12/26/2020 16:51:14","exceptionDetails":{"ErrorMessage":"Enumerator failed to MoveNextAsync.","OriginatingClass":"Microsoft.EntityFrameworkCore.Query.ShapedQueryCompilingExpressionVisitor+<SingleAsync>d__21`1","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void MoveNext()"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/appointments/GetPatientAppointmentsToday","ControllerMethodName":"GetPatientAppointmentsToday","ControllerName":"Appointments"}}
{"occurAt":"12/26/2020 16:51:42","exceptionDetails":{"ErrorMessage":"Enumerator failed to MoveNextAsync.","OriginatingClass":"Microsoft.EntityFrameworkCore.Query.ShapedQueryCompilingExpressionVisitor+<SingleAsync>d__21`1","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Void MoveNext()"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/appointments/GetPatientAppointmentsToday","ControllerMethodName":"GetPatientAppointmentsToday","ControllerName":"Appointments"}}
{"occurAt":"12/27/2020 18:41:15","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/CheckIn/GetCheckedInPatient","ControllerMethodName":"GetCheckedInPatient","ControllerName":"CheckIn"}}
{"occurAt":"12/27/2020 18:41:30","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/appointments/statuses","ControllerMethodName":"GetAppointmentStatuses","ControllerName":"Appointments"}}
{"occurAt":"12/27/2020 18:41:30","exceptionDetails":{"ErrorMessage":"A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)","OriginatingClass":"Microsoft.Data.ProviderBase.DbConnectionPool","FileName":null,"LineNumber":0,"ColumnNumber":0,"MethodThatThrowException":"Boolean TryGetConnection(System.Data.Common.DbConnection, UInt32, Boolean, Boolean, Microsoft.Data.Common.DbConnectionOptions, Microsoft.Data.ProviderBase.DbConnectionInternal ByRef)"},"RequestDetails":{"HttpMethod":"GET","Path":"/api/locations","ControllerMethodName":"GetLocations","ControllerName":"Locations"}}